vomsimporter.py

```diff
@@ -571,16 +572,45 @@ def import_voms_user(self, voms_user):
                     voms_user, iam_user, cern_login)
 
     def resolve_cern_login_from_ldap(self, voms_user):
-        cern_login = subprocess.check_output(["resolve_cern_login", str(voms_user[
-            'cernHrId']), self._ldap_host, self._ldap_port]).replace("\n", "").strip()
+        cern_login = None
+        lfilter = "(&(objectClass=user)(employeeType=Primary)(employeeID={0}))".format(voms_user['cernHrId'])
```

Comment:
In theory when we arrive here `voms_user['cernHrId']` exists, but practice is different. So I suggest to set specifically a variable to `voms_user['cernHrId']` and check for `None`. That var can be used in the rest of the function.